### ISSUE TITLE
[Youtube] Text to Speech read incorrect values for Super Chats

### ISSUE THREAD

**Issue by diegoveloper** [steveseguin/social_stream#25]
Firstly, thanks for this awesome tool! My audience and I are enjoying this. I wanted to try the text-to-speech feature for super chats and I think we found a bug. The bot was speaking some HTML code.

I'll paste the HTML code captured from the CHAT. 

For super chats without message
```HTML
<yt-live-chat-paid-message-renderer class="style-scope yt-live-chat-item-list-renderer" id="ChwKGkNPakIzTnpQb19rQ0ZZaXU1UWNkMTY4RlZn" has-inline-action-buttons="2" show-only-header="" allow-animations="" style="--yt-live-chat-paid-message-primary-color:rgba(30,136,229,1); --yt-live-chat-paid-message-secondary-color:rgba(21,101,192,1); --yt-live-chat-paid-message-header-color:rgba(255,255,255,1); --yt-live-chat-paid-message-author-name-color:rgba(255,255,255,0.701961); --yt-live-chat-paid-message-timestamp-color:rgba(255,255,255,0.501961); --yt-live-chat-paid-message-color:rgba(255,255,255,1);"><!--css-build:shady--><div id="card" class="style-scope yt-live-chat-paid-message-renderer">
  <div id="header" class="style-scope yt-live-chat-paid-message-renderer">
    
      <yt-img-shadow id="author-photo" height="40" width="40" class="style-scope yt-live-chat-paid-message-renderer no-transition" loaded="" style="background-color: transparent;"><!--css-build:shady--><img id="img" draggable="false" class="style-scope yt-img-shadow" alt="" height="40" width="40" src="https://yt4.ggpht.com/ytc/AMLnZu_-fyZa-Utg71gtg26skKKuN7aatK2J_TTSKF3JEQ=s64-c-k-c0x00ffffff-no-rj"></yt-img-shadow>
    <dom-if restamp="" class="style-scope yt-live-chat-paid-message-renderer"><template is="dom-if"></template></dom-if>
    <dom-if class="style-scope yt-live-chat-paid-message-renderer"><template is="dom-if"></template></dom-if>
    <dom-if restamp="" class="style-scope yt-live-chat-paid-message-renderer"><template is="dom-if"></template></dom-if>
    <div id="header-content" class="style-scope yt-live-chat-paid-message-renderer">
      <div id="header-content-primary-column" class="style-scope yt-live-chat-paid-message-renderer">
        <div id="author-name-chip" class="style-scope yt-live-chat-paid-message-renderer">
             <yt-live-chat-author-chip disable-highlighting="" class="style-scope yt-live-chat-paid-message-renderer"><!--css-build:shady--><span id="author-name" dir="auto" class="style-scope yt-live-chat-author-chip">Arturo Espinosa<span id="chip-badges" class="style-scope yt-live-chat-author-chip"></span></span><span id="chat-badges" class="style-scope yt-live-chat-author-chip"></span></yt-live-chat-author-chip>
        </div>
        <div id="purchase-amount-column" class="style-scope yt-live-chat-paid-message-renderer">
          <yt-img-shadow id="currency-img" height="16" width="16" class="style-scope yt-live-chat-paid-message-renderer no-transition" hidden=""><!--css-build:shady--><img id="img" draggable="false" class="style-scope yt-img-shadow" alt="" height="16" width="16"></yt-img-shadow>
          <div id="purchase-amount" class="style-scope yt-live-chat-paid-message-renderer">
            <yt-formatted-string class="style-scope yt-live-chat-paid-message-renderer">MX$10.00</yt-formatted-string>
          </div>
        </div>
      </div>
      <span id="timestamp" class="style-scope yt-live-chat-paid-message-renderer">5:58</span>
    </div>
    <div id="menu" class="style-scope yt-live-chat-paid-message-renderer">
      <yt-icon-button id="menu-button" class="style-scope yt-live-chat-paid-message-renderer"><!--css-build:shady--><button id="button" class="style-scope yt-icon-button" aria-label="Chat actions">
        <yt-icon icon="more_vert" class="style-scope yt-live-chat-paid-message-renderer"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope yt-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope yt-icon"><path d="M12,16.5c0.83,0,1.5,0.67,1.5,1.5s-0.67,1.5-1.5,1.5s-1.5-0.67-1.5-1.5S11.17,16.5,12,16.5z M10.5,12 c0,0.83,0.67,1.5,1.5,1.5s1.5-0.67,1.5-1.5s-0.67-1.5-1.5-1.5S10.5,11.17,10.5,12z M10.5,6c0,0.83,0.67,1.5,1.5,1.5 s1.5-0.67,1.5-1.5S12.83,4.5,12,4.5S10.5,5.17,10.5,6z" class="style-scope yt-icon"></path></g></svg><!--css-build:shady--></yt-icon>
      </button><yt-interaction id="interaction" class="circular style-scope yt-icon-button"><!--css-build:shady--><div class="stroke style-scope yt-interaction"></div><div class="fill style-scope yt-interaction"></div></yt-interaction></yt-icon-button>
    </div>
  </div>
  <div id="content" class="style-scope yt-live-chat-paid-message-renderer">
    <div id="message" dir="auto" class="style-scope yt-live-chat-paid-message-renderer"></div>
    <div id="input-container" class="style-scope yt-live-chat-paid-message-renderer">
      <dom-if class="style-scope yt-live-chat-paid-message-renderer"><template is="dom-if"></template></dom-if>
    </div>
    <yt-formatted-string id="deleted-state" class="style-scope yt-live-chat-paid-message-renderer"><!--css-build:shady--></yt-formatted-string>
    <div id="footer" class="style-scope yt-live-chat-paid-message-renderer"></div>
  </div>
</div>
<div id="buy-flow-button" class="style-scope yt-live-chat-paid-message-renderer" hidden=""></div>
<div id="inline-action-button-container" class="style-scope yt-live-chat-paid-message-renderer" aria-hidden="true">
  <div id="inline-action-buttons" class="style-scope yt-live-chat-paid-message-renderer"><yt-button-renderer class="style-scope yt-live-chat-paid-message-renderer style-default size-default" is-icon-button="" has-no-text=""><a class="yt-simple-endpoint style-scope yt-button-renderer" tabindex="-1"><yt-icon-button id="button" class="style-scope yt-button-renderer style-default size-default"><!--css-build:shady--><button id="button" class="style-scope yt-icon-button" aria-label="Remove"><yt-icon class="style-scope yt-button-renderer"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope yt-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope yt-icon"><path d="M11,17H9V8h2V17z M15,8h-2v9h2V8z M19,4v1h-1v16H6V5H5V4h4V3h6v1H19z M17,5H7v15h10V5z" class="style-scope yt-icon"></path></g></svg><!--css-build:shady--></yt-icon></button><yt-interaction id="interaction" class="circular style-scope yt-icon-button"><!--css-build:shady--><div class="stroke style-scope yt-interaction"></div><div class="fill style-scope yt-interaction"></div></yt-interaction></yt-icon-button></a></yt-button-renderer><yt-button-renderer class="style-scope yt-live-chat-paid-message-renderer style-default size-default" is-icon-button="" has-no-text=""><a class="yt-simple-endpoint style-scope yt-button-renderer" tabindex="-1"><yt-icon-button id="button" class="style-scope yt-button-renderer style-default size-default"><!--css-build:shady--><button id="button" class="style-scope yt-icon-button" aria-label="Hide user on this channel"><yt-icon class="style-scope yt-button-renderer"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope yt-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope yt-icon"><path d="M12,3c-4.96,0-9,4.04-9,9s4.04,9,9,9s9-4.04,9-9S16.96,3,12,3 M12,2c5.52,0,10,4.48,10,10s-4.48,10-10,10S2,17.52,2,12 S6.48,2,12,2L12,2z M19,13H5v-2h14V13z" class="style-scope yt-icon"></path></g></svg><!--css-build:shady--></yt-icon></button><yt-interaction id="interaction" class="circular style-scope yt-icon-button"><!--css-build:shady--><div class="stroke style-scope yt-interaction"></div><div class="fill style-scope yt-interaction"></div></yt-interaction></yt-icon-button></a></yt-button-renderer></div>
</div>
</yt-live-chat-paid-message-renderer>
```
Output 

(chat window)
![Screen Shot 2022-07-31 at 5 25 07 PM](https://user-images.githubusercontent.com/4898256/182047980-0cec677a-890d-4a80-ba7e-71c64dee23d3.png)

(obs)
![Screen Shot 2022-07-31 at 5 25 00 PM](https://user-images.githubusercontent.com/4898256/182048300-61eb3373-d608-4f91-92b0-1f0a663803dd.png)

For super chats with message

```HTML
<div id="card" class="style-scope yt-live-chat-paid-message-renderer">
  <div id="header" class="style-scope yt-live-chat-paid-message-renderer">
    
      <yt-img-shadow id="author-photo" height="40" width="40" class="style-scope yt-live-chat-paid-message-renderer no-transition" loaded="" style="background-color: transparent;"><!--css-build:shady--><img id="img" draggable="false" class="style-scope yt-img-shadow" alt="" height="40" width="40" src="https://yt4.ggpht.com/ytc/AMLnZu8-Z8igOdnnyah1rjep-m3DgzgBJdfjKaH0CUPR6w=s64-c-k-c0x00ffffff-no-rj"></yt-img-shadow>
    <dom-if restamp="" class="style-scope yt-live-chat-paid-message-renderer"><template is="dom-if"></template></dom-if>
    <dom-if class="style-scope yt-live-chat-paid-message-renderer"><template is="dom-if"></template></dom-if>
    <dom-if restamp="" class="style-scope yt-live-chat-paid-message-renderer"><template is="dom-if"></template></dom-if>
    <div id="header-content" class="style-scope yt-live-chat-paid-message-renderer">
      <div id="header-content-primary-column" class="style-scope yt-live-chat-paid-message-renderer">
        <div id="author-name-chip" class="style-scope yt-live-chat-paid-message-renderer">
             <yt-live-chat-author-chip disable-highlighting="" class="style-scope yt-live-chat-paid-message-renderer"><!--css-build:shady--><span id="author-name" dir="auto" class="style-scope yt-live-chat-author-chip">Andres Agredo<span id="chip-badges" class="style-scope yt-live-chat-author-chip"></span></span><span id="chat-badges" class="style-scope yt-live-chat-author-chip"></span></yt-live-chat-author-chip>
        </div>
        <div id="purchase-amount-column" class="style-scope yt-live-chat-paid-message-renderer">
          <yt-img-shadow id="currency-img" height="16" width="16" class="style-scope yt-live-chat-paid-message-renderer no-transition" hidden=""><!--css-build:shady--><img id="img" draggable="false" class="style-scope yt-img-shadow" alt="" height="16" width="16"></yt-img-shadow>
          <div id="purchase-amount" class="style-scope yt-live-chat-paid-message-renderer">
            <yt-formatted-string class="style-scope yt-live-chat-paid-message-renderer">$5.00</yt-formatted-string>
          </div>
        </div>
      </div>
      <span id="timestamp" class="style-scope yt-live-chat-paid-message-renderer">1:39:18</span>
    </div>
    <div id="menu" class="style-scope yt-live-chat-paid-message-renderer">
      <yt-icon-button id="menu-button" class="style-scope yt-live-chat-paid-message-renderer"><!--css-build:shady--><button id="button" class="style-scope yt-icon-button" aria-label="Chat actions">
        <yt-icon icon="more_vert" class="style-scope yt-live-chat-paid-message-renderer"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope yt-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope yt-icon"><path d="M12,16.5c0.83,0,1.5,0.67,1.5,1.5s-0.67,1.5-1.5,1.5s-1.5-0.67-1.5-1.5S11.17,16.5,12,16.5z M10.5,12 c0,0.83,0.67,1.5,1.5,1.5s1.5-0.67,1.5-1.5s-0.67-1.5-1.5-1.5S10.5,11.17,10.5,12z M10.5,6c0,0.83,0.67,1.5,1.5,1.5 s1.5-0.67,1.5-1.5S12.83,4.5,12,4.5S10.5,5.17,10.5,6z" class="style-scope yt-icon"></path></g></svg><!--css-build:shady--></yt-icon>
      </button><yt-interaction id="interaction" class="circular style-scope yt-icon-button"><!--css-build:shady--><div class="stroke style-scope yt-interaction"></div><div class="fill style-scope yt-interaction"></div></yt-interaction></yt-icon-button>
    </div>
  </div>
  <div id="content" class="style-scope yt-live-chat-paid-message-renderer">
    <div id="message" dir="auto" class="style-scope yt-live-chat-paid-message-renderer">Saludos Diego</div>
    <div id="input-container" class="style-scope yt-live-chat-paid-message-renderer">
      <dom-if class="style-scope yt-live-chat-paid-message-renderer"><template is="dom-if"></template></dom-if>
    </div>
    <yt-formatted-string id="deleted-state" class="style-scope yt-live-chat-paid-message-renderer"><!--css-build:shady--></yt-formatted-string>
    <div id="footer" class="style-scope yt-live-chat-paid-message-renderer"></div>
  </div>
</div>
```

Output

(chat window)
![Screen Shot 2022-07-31 at 5 42 59 PM](https://user-images.githubusercontent.com/4898256/182048308-1d967d9b-a75a-42c4-9a82-d18ba452112d.png)

(obs)
![Screen Shot 2022-07-31 at 5 27 21 PM](https://user-images.githubusercontent.com/4898256/182048015-1258d153-4c93-4fd7-b385-68ce3bed6dcd.png)

What I could hear from this last chat was something like this: "Donator Name"  `div class = donation yt-formatted-string class ... message renderer`   "5 dollars"  `yt-formatter String div`  "Message here".

Also if a small "pause" could be added after the Donator/Author name it would be great! :D

**Comment by steveseguin** [comment]
Thank you for reporting the issue!  I'll see about getting a fix out tonight or tomorrow latest.

**Comment by diegoveloper** [comment]
hey @steveseguin sorry to bother, this is just a question, I know that it's possible to enable the text to speech for Single message overlay, and I can configure when there is a super chat , display the single message (with text to speech), but is there an option to "disable the text-speech" when the Message is selected from the dockable stream chat and displayed on the Single message overlay? 

In summary: Activate the text to speech on Single message overlay only when it comes from a super chat message and not when we click any non-superchat message?

**Comment by steveseguin** [comment]
so, to confirm, you want an option for text to speech to only trigger on single message overlays that contain a Donation?

I'll need to add that in, as I don't have that as an option currently.  Shouldn't be hard to add though, so I'll see about getting something up soon

**Comment by diegoveloper** [comment]
> so, to confirm, you want an option for text to speech to only trigger on single message overlays that contain a Donation?
> 
> I'll need to add that in, as I don't have that as an option currently. Shouldn't be hard to add though, so I'll see about getting something up soon

Yes! , thank you!

**Comment by steveseguin** [comment]
okay, I pushed an option to github.

&ttsdonos

Let me know if it doesn't work.  You may need to hard refresh the browser for it to apply.

**Comment by diegoveloper** [comment]
@steveseguin one quick question, how do you usually update the code? do we need to overwrite? I say this because I removed the option to read data from whatsapp and I had to update the file `manifest.json`, if I overwrite everything I'll need to update that file again, or is there any other way to ignore some chats? I'm only interested in youtube for now.

Thanks

**Comment by steveseguin** [comment]
I added WhatsApp as a toggle. It is now off by default.

![image](https://user-images.githubusercontent.com/2575698/182515459-aa8569b4-6727-46b2-833d-5ef42e8274f3.png)

You should be able to just replace the files and not need to edit things anymore.

**Comment by diegoveloper** [comment]
awesome! thank you

**Comment by diegoveloper** [comment]
I was testing the donation message using the "Fake a comment for testing" and this still says some HTML text:

"TEST MSG "  `<div class = donation ytube donation> ...` "100 gold" "A test message".

![Screen Shot 2022-08-02 at 10 16 22 PM](https://user-images.githubusercontent.com/4898256/182516555-4beb91e1-a370-4726-a3f5-4b14ceac9e2a.png)

Btw, what's this  "V" button? 
![Screen Shot 2022-08-02 at 10 18 37 PM](https://user-images.githubusercontent.com/4898256/182516766-bbb7ec2a-ce6d-4625-9cfc-f0c56d35a8d6.png)

**Comment by steveseguin** [comment]
I just pushed an update; should kick in automatically in a few minutes.  Sorry, hopefully it will be working better now?  I wasn't able to reproduce the issue in my testing, but I think I see what might be causing it.   

 What system / browser are  you using by chance? Whatever language parser your system/browser is using doesn't seem to be as robust as what I've been testing with it seems. 
 
as for that little down arrow, it will lock the scroll to the bottom of the page, so if you have a problem with the page not scrolling down automatically, press that should lock it at the bottom.  You'll need to  unlock it if you want to scroll up thru the messages.

**Comment by diegoveloper** [comment]
hi @steveseguin I tested the latest changes and it works! , just a minor issue, when the bot is speaking, there is no pause between the Name and the Donation.   "Jhon Doe100 gold" <pause> "this is a test message".  Also it would be great if you add a new button like "Fake a super chat comment for testing", I think now is random (normal message and super chat message for testing), right?

**Comment by diegoveloper** [comment]
thanks @steveseguin , this works perfect, I just noticed there is an extra `letter` after the donation amount "100 gold`<dot>`" or "100 gold." but I don't mind , thank you very much!

**Comment by steveseguin** [comment]
If you let me know what language + operating system + browser you're using, I can test for it in the future.  I don't have the issues you mention, but I'm glad it's acceptable now.

**Comment by diegoveloper** [comment]
> If you let me know what language + operating system + browser you're using, I can test for it in the future. I don't have the issues you mention, but I'm glad it's acceptable now.

Sure, I'm using MacOs 12.5 Monterey, this is the browser : Chrome Version 104.0.5112.79 (Official Build) (arm64) and I'm using the default language English as default but I set the text to speech to : ES-PE  (spanish - Peru)

**Comment by steveseguin** [comment]
Wonderful. thank you.

**Comment by diegoveloper** [comment]
> Wonderful. thank you.

hi @steveseguin I couldn't replicate the issue using another `ES` voice, like "voice=Jorge" works fine for me, the issue was with "voice=Diego" , not sure why that voice add "dot", after the donation amount.

Btw I was using `es-ES` but it was taking the first `es` voice = "Diego" from ARG -> `es-AR`

```javascript
0: SpeechSynthesisVoice {voiceURI: 'Alex', name: 'Alex', lang: 'en-US', localService: true, default: true}
1: SpeechSynthesisVoice {voiceURI: 'Alice', name: 'Alice', lang: 'it-IT', localService: true, default: false}
2: SpeechSynthesisVoice {voiceURI: 'Alva', name: 'Alva', lang: 'sv-SE', localService: true, default: false}
3: SpeechSynthesisVoice {voiceURI: 'Amelie', name: 'Amelie', lang: 'fr-CA', localService: true, default: false}
4: SpeechSynthesisVoice {voiceURI: 'Anna', name: 'Anna', lang: 'de-DE', localService: true, default: false}
5: SpeechSynthesisVoice {voiceURI: 'Carmit', name: 'Carmit', lang: 'he-IL', localService: true, default: false}
6: SpeechSynthesisVoice {voiceURI: 'Damayanti', name: 'Damayanti', lang: 'id-ID', localService: true, default: false}
7: SpeechSynthesisVoice {voiceURI: 'Daniel', name: 'Daniel', lang: 'en-GB', localService: true, default: false}
8: SpeechSynthesisVoice {voiceURI: 'Diego', name: 'Diego', lang: 'es-AR', localService: true, default: false}
9: SpeechSynthesisVoice {voiceURI: 'Ellen', name: 'Ellen', lang: 'nl-BE', localService: true, default: false}
10: SpeechSynthesisVoice {voiceURI: 'Fiona', name: 'Fiona', lang: 'en', localService: true, default: false}
11: SpeechSynthesisVoice {voiceURI: 'Fred', name: 'Fred', lang: 'en-US', localService: true, default: false}
12: SpeechSynthesisVoice {voiceURI: 'Ioana', name: 'Ioana', lang: 'ro-RO', localService: true, default: false}
13: SpeechSynthesisVoice {voiceURI: 'Joana', name: 'Joana', lang: 'pt-PT', localService: true, default: false}
14: SpeechSynthesisVoice {voiceURI: 'Jorge', name: 'Jorge', lang: 'es-ES', localService: true, default: false}
15: SpeechSynthesisVoice {voiceURI: 'Juan', name: 'Juan', lang: 'es-MX', localService: true, default: false}
16: SpeechSynthesisVoice {voiceURI: 'Kanya', name: 'Kanya', lang: 'th-TH', localService: true, default: false}
17: SpeechSynthesisVoice {voiceURI: 'Karen', name: 'Karen', lang: 'en-AU', localService: true, default: false}
18: SpeechSynthesisVoice {voiceURI: 'Kyoko', name: 'Kyoko', lang: 'ja-JP', localService: true, default: false}
19: SpeechSynthesisVoice {voiceURI: 'Laura', name: 'Laura', lang: 'sk-SK', localService: true, default: false}
20: SpeechSynthesisVoice {voiceURI: 'Lekha', name: 'Lekha', lang: 'hi-IN', localService: true, default: false}
21: SpeechSynthesisVoice {voiceURI: 'Luca', name: 'Luca', lang: 'it-IT', localService: true, default: false}
22: SpeechSynthesisVoice {voiceURI: 'Luciana', name: 'Luciana', lang: 'pt-BR', localService: true, default: false}
23: SpeechSynthesisVoice {voiceURI: 'Maged', name: 'Maged', lang: 'ar-SA', localService: true, default: false}
24: SpeechSynthesisVoice {voiceURI: 'Mariska', name: 'Mariska', lang: 'hu-HU', localService: true, default: false}
25: SpeechSynthesisVoice {voiceURI: 'Mei-Jia', name: 'Mei-Jia', lang: 'zh-TW', localService: true, default: false}
26: SpeechSynthesisVoice {voiceURI: 'Melina', name: 'Melina', lang: 'el-GR', localService: true, default: false}
27: SpeechSynthesisVoice {voiceURI: 'Milena', name: 'Milena', lang: 'ru-RU', localService: true, default: false}
28: SpeechSynthesisVoice {voiceURI: 'Moira', name: 'Moira', lang: 'en-IE', localService: true, default: false}
29: SpeechSynthesisVoice {voiceURI: 'Monica', name: 'Monica', lang: 'es-ES', localService: true, default: false}
30: SpeechSynthesisVoice {voiceURI: 'Nora', name: 'Nora', lang: 'nb-NO', localService: true, default: false}
31: SpeechSynthesisVoice {voiceURI: 'Paulina', name: 'Paulina', lang: 'es-MX', localService: true, default: false}
32: SpeechSynthesisVoice {voiceURI: 'Rishi', name: 'Rishi', lang: 'en-IN', localService: true, default: false}
33: SpeechSynthesisVoice {voiceURI: 'Samantha', name: 'Samantha', lang: 'en-US', localService: true, default: false}
34: SpeechSynthesisVoice {voiceURI: 'Sara', name: 'Sara', lang: 'da-DK', localService: true, default: false}
35: SpeechSynthesisVoice {voiceURI: 'Satu', name: 'Satu', lang: 'fi-FI', localService: true, default: false}
36: SpeechSynthesisVoice {voiceURI: 'Sin-ji', name: 'Sin-ji', lang: 'zh-HK', localService: true, default: false}
37: SpeechSynthesisVoice {voiceURI: 'Tessa', name: 'Tessa', lang: 'en-ZA', localService: true, default: false}
38: SpeechSynthesisVoice {voiceURI: 'Thomas', name: 'Thomas', lang: 'fr-FR', localService: true, default: false}
39: SpeechSynthesisVoice {voiceURI: 'Ting-Ting', name: 'Ting-Ting', lang: 'zh-CN', localService: true, default: false}
40: SpeechSynthesisVoice {voiceURI: 'Veena', name: 'Veena', lang: 'en-IN', localService: true, default: false}
41: SpeechSynthesisVoice {voiceURI: 'Victoria', name: 'Victoria', lang: 'en-US', localService: true, default: false}
42: SpeechSynthesisVoice {voiceURI: 'Xander', name: 'Xander', lang: 'nl-NL', localService: true, default: false}
43: SpeechSynthesisVoice {voiceURI: 'Yelda', name: 'Yelda', lang: 'tr-TR', localService: true, default: false}
44: SpeechSynthesisVoice {voiceURI: 'Yuna', name: 'Yuna', lang: 'ko-KR', localService: true, default: false}
45: SpeechSynthesisVoice {voiceURI: 'Yuri', name: 'Yuri', lang: 'ru-RU', localService: true, default: false}
46: SpeechSynthesisVoice {voiceURI: 'Zosia', name: 'Zosia', lang: 'pl-PL', localService: true, default: false}
47: SpeechSynthesisVoice {voiceURI: 'Zuzana', name: 'Zuzana', lang: 'cs-CZ', localService: true, default: false}
48: SpeechSynthesisVoice {voiceURI: 'Google Deutsch', name: 'Google Deutsch', lang: 'de-DE', localService: false, default: false}
49: SpeechSynthesisVoice {voiceURI: 'Google US English', name: 'Google US English', lang: 'en-US', localService: false, default: false}
50: SpeechSynthesisVoice {voiceURI: 'Google UK English Female', name: 'Google UK English Female', lang: 'en-GB', localService: false, default: false}
51: SpeechSynthesisVoice {voiceURI: 'Google UK English Male', name: 'Google UK English Male', lang: 'en-GB', localService: false, default: false}
52: SpeechSynthesisVoice {voiceURI: 'Google español', name: 'Google español', lang: 'es-ES', localService: false, default: false}
53: SpeechSynthesisVoice {voiceURI: 'Google español de Estados Unidos', name: 'Google español de Estados Unidos', lang: 'es-US', localService: false, default: false}
54: SpeechSynthesisVoice {voiceURI: 'Google français', name: 'Google français', lang: 'fr-FR', localService: false, default: false}
55: SpeechSynthesisVoice {voiceURI: 'Google हिन्दी', name: 'Google हिन्दी', lang: 'hi-IN', localService: false, default: false}
56: SpeechSynthesisVoice {voiceURI: 'Google Bahasa Indonesia', name: 'Google Bahasa Indonesia', lang: 'id-ID', localService: false, default: false}
57: SpeechSynthesisVoice {voiceURI: 'Google italiano', name: 'Google italiano', lang: 'it-IT', localService: false, default: false}
58: SpeechSynthesisVoice {voiceURI: 'Google 日本語', name: 'Google 日本語', lang: 'ja-JP', localService: false, default: false}
59: SpeechSynthesisVoice {voiceURI: 'Google 한국의', name: 'Google 한국의', lang: 'ko-KR', localService: false, default: false}
60: SpeechSynthesisVoice {voiceURI: 'Google Nederlands', name: 'Google Nederlands', lang: 'nl-NL', localService: false, default: false}
61: SpeechSynthesisVoice {voiceURI: 'Google polski', name: 'Google polski', lang: 'pl-PL', localService: false, default: false}
62: SpeechSynthesisVoice {voiceURI: 'Google português do Brasil', name: 'Google português do Brasil', lang: 'pt-BR', localService: false, default: false}
63: SpeechSynthesisVoice {voiceURI: 'Google русский', name: 'Google русский', lang: 'ru-RU', localService: false, default: false}
64: SpeechSynthesisVoice {voiceURI: 'Google 普通话（中国大陆）', name: 'Google 普通话（中国大陆）', lang: 'zh-CN', localService: false, default: false}
65: SpeechSynthesisVoice {voiceURI: 'Google 粤語（香港）', name: 'Google 粤語（香港）', lang: 'zh-HK', localService: false, default: false}
66: SpeechSynthesisVoice {voiceURI: 'Google 國語（臺灣）', name: 'Google 國語（臺灣）', lang: 'zh-TW', localService: false, default: false}
```

**Comment by steveseguin** [comment]
Thank you. That might be just the test message; I think I fixed it in the
actual YouTube component.

I'll fix the test message in the morning

Thank you for reporting it.
